### PR TITLE
Functional demo

### PR DESCRIPTION
**Pull request by theFido**
ℹ️ First draft, for now this should be enough to import the lib and have a basic behavior. If you have cool ideas to make it do something meaningful, lmk 👂

### CODE REVIEW

File: lib.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+// Name returns the library name
+func Name() string {
+	return "sharedlib"
+}
+
+// Hi let's be friends
+func Hi(name string) string {
+	return "👋  Hi " + name
+}
```
Comment (kmhagan) [inline lib.go:20]: thoughts: i _feel_ like we should try to end with a newline in every file

**Review by kmhagan — APPROVED** [review]

File: lib.go (hunk 1 above)
Comment (timjonesdev) [reply lib.go:20]: does `go fmt` enforce that? If so, we should add it as an action!

**Review by timjonesdev — APPROVED** [review]
lgtm

File: lib.go (hunk 1 above)
Comment (theFido) [reply lib.go:20]: I can/will add it with .editorconfig, I always customize my projects with it to complement `goimports`

**Pull request merged**